### ISSUE TITLE
Add config.hosts instructions to GitPod Installation docs

### ISSUE THREAD

**Issue by Amorpheuz** [thepracticaldev/dev.to#8458]
**Describe the bug**
Opening DEV in Gitpod from with https://gitpod.io/#https://github.com/thepracticaldev/dev.to leads to a `Blocked Host` error after a successful install of everything and `bin/startup` (Screenshot below). This is due to `Host Authorization` in Rails 6 where `config.hosts` needs to be configured. 

It would be nice to have a helpful indication in the GitPod docs about this process and where to place the line. A new entry needs to be created in `application.yml` with `APP_DOMAN: <URL>` where the URL is the one mentioned in the error message within `" "`. Not sure if the GitPod file can be configured to do this automatically or not.

**To Reproduce**

1. Open https://gitpod.io/#https://github.com/thepracticaldev/dev.to and wait for installation and `bin/startup` to complete.
2. Open the Browser Preview tab if it doesn't open automatically.
3. A similar error with `Blocked Host` will be printed.

**Expected behavior**
Either the `APP_DOMAN` env var is automatically configured or there are instructions printed in docs and in GitPod console to do the task before `bin/startup` runs.

**Screenshots**
![image](https://user-images.githubusercontent.com/22113778/84563591-51e81100-ad7a-11ea-8be5-a9aa66dce0ac.png)

**Additional context**
Should perform a checkup if the process can be automated.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
Hi @Amorpheuz, thanks for the issue. Last week I started working on making this automatic, I'll try to finish it soon.
